README.md

```diff
@@ -14,27 +14,30 @@ If you are interested in learning more about this project, start with reading [P
 Setup for developers (Unix)
 ---------------------------
 
-1. Make sure you have installed Python 3.6, [pip3](https://pip.pypa.io/en/latest/) and [virtualenv](http://www.virtualenv.org/en/latest/).
-1. If working behind a proxy, make sure your environment variables are properly set up. If
+1. Make sure you have installed Python 3.6 or 3.7, [pip3](https://pip.pypa.io/en/latest/) and [virtualenv](http://www.virtualenv.org/en/latest/).
+2. If working behind a proxy, make sure your environment variables are properly set up. If
    you still get an error due to proxy, use "-E" flag along with "sudo" to export all the
    environment variables.
-1. Make sure you have python3-dev installed on your operating system. For Debian, you would additionally require libpq-dev.
+3. Make sure you have python3-dev installed on your operating system. For Debian, you would additionally require libpq-dev.
    Install by using `sudo apt-get install libpq-dev python3-dev`
```

Comment:
Do we need this package `libpq-dev python3-dev`, because I have noticed while working with GitHub actions's setup it is not required (or not addtionally required for adding this package) [see](https://github.com/anitab-org/portal/pull/632/files#diff-90829e76e906f1c73140c7ded7e1b268).

Reply:
I definitely think we need this because `psycopg2` is a C-extension module, so it needs the header files for linking.

Are you sure about this?

Reply:
I think we need this 🙄

Reply:
@satya7289 ^^